servers/register_server_types.cpp

```diff
@@ -110,7 +110,7 @@ void preregister_server_types() {
 	shader_types = memnew(ShaderTypes);
 
 	GDREGISTER_CLASS(TextServerManager);
-	GDREGISTER_VIRTUAL_CLASS(TextServer);
+	GDREGISTER_ABSTRACT_CLASS(TextServer);
 	GDREGISTER_CLASS(TextServerExtension);
```

Comment:
Another candidate for merging `TextServerExtension` into a virtual `TextServer` I guess @bruvzg.

Reply:
This make sense, but last time I have tried to do so, every `GDVIRTUAL` binding which is using enums for the `TextServer` stopped working, seems like registering enums and binding virtual methods should have a certain order. So it might need some extra changes.

Reply:
It also sounds like @reduz suggests keeping `*Extension` classes for classes where *a lot* of the API is virtual, so this might be the case here.

Reply:
I think for the large ones (like script, physics server, etc. Using Extension makes more sense because otherwise the code gets to be a mess.